sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreSuite.scala

```diff
@@ -189,12 +232,15 @@ class RocksDBStateStoreSuite extends StateStoreSuiteBase[RocksDBStateStoreProvid
       numColsPrefixKey: Int,
       sqlConf: Option[SQLConf] = None,
       conf: Configuration = new Configuration,
-      useColumnFamilies: Boolean = false): RocksDBStateStoreProvider = {
+      useColumnFamilies: Boolean = false,
+      useMultipleValuesPerKey: Boolean = false): RocksDBStateStoreProvider = {
     val provider = new RocksDBStateStoreProvider()
     provider.init(
       storeId, keySchema, valueSchema, numColsPrefixKey = numColsPrefixKey,
       useColumnFamilies,
-      new StateStoreConf(sqlConf.getOrElse(SQLConf.get)), conf)
+      new StateStoreConf(sqlConf.getOrElse(SQLConf.get)), conf,
+      useMultipleValuesPerKey
```

Comment:
Nit: indent ?

Reply:
The indentation is actually fine here. `useMultipleValuesPerKey` is not a parameter for `StateStoreConf`. However, I think it would be more readable if all params are in a new line, so I have made that change.